### DISCUSSION TITLE
Deployment issue and error in my terminal

### DISCUSSION THREAD

**Discussion by deeneycode** [Cyfrin/foundry-full-course-cu#2509] [Q&A]
im trying to deploy my contract below is the error shpwing in my terminal:
No files changed, compilation skipped
2024-09-20T08:33:30.260016Z ERROR foundry_compilers_artifacts_solc::sources: error="/home/deeney/foundry-f23/foundry-fund-me/script/FundMe.deploy.sol": No such file or directory (os error 2)
Error: 
failed to read artifact source file for `/home/deeney/foundry-f23/foundry-fund-me/script/FundMe.deploy.sol`

Context:
- "/home/deeney/foundry-f23/foundry-fund-me/script/FundMe.deploy.sol": No such file or directory (os error 2)

**Comment by EngrPips** [comment]
Hello @deeneycode, Can you please show your working directory, as I have shown mine below?

<img width="306" alt="Screenshot 2024-09-20 at 15 50 25" src="https://github.com/user-attachments/assets/c5971779-66ab-42e0-a808-818d188da912">

**Reply by deeneycode** [reply]
Have been able to resolve the issue @EngrPips using forge clean (you suggested that om a post thou)

**Reply by EngrPips** [reply]
That is interesting. I am glad you unblocked yourself. Keep crushing it.
